arch/cpu/gecko/efr32bg22c224f512im40/Makefile.efr32bg22c224f512im40

```diff
@@ -0,0 +1,16 @@
+include $(CONTIKI_CPU)/Makefile.gecko_sdk  
+
+DEVICE_FAMILY := EFR32BG22
+RAIL_CPU := efr32xg22
+RAIL_FAMILY := efr32xg2x
+SL_SERIES := 2
+
+CFLAGS += -mfpu=fpv5-sp-d16 -mfloat-abi=hard -mcmse
```

Comment:
Is this always enabling TrustZone?

`-mcmse` is already added by `Makefile.arm` when `TRUSTZONE_SECURE_BUILD = 1`.